Helper/Cart.php

```diff
@@ -1341,7 +1341,7 @@ function ($item) use ($imageHelper, &$totalAmount, &$diff, $storeId, $currencyCo
                 $unitPrice   = $item->getCalculationPrice();
                 $itemTotalAmount = $unitPrice * $item->getQty();
 
-                $roundedTotalAmount = CurrencyUtils::toMinor($itemTotalAmount, $currencyCode);
+                $roundedTotalAmount = CurrencyUtils::toMinor($unitPrice, $currencyCode) * round($item->getQty());
```

Comment:
@johnmccombs1 Do you know why we need to round item Qty? I see we have the same code below.

Reply:
@vitaliyreznikov On M2, we rounded the quantity when sending to Bolt. See this: https://github.com/BoltApp/bolt-magento2/blob/master/Helper/Cart.php#L1361 So we should also round item qty when calculating the total amount.